README.md

```diff
@@ -92,21 +92,23 @@ Inside your `AppDelegate.application(_, handleEventsForBackgroundURLSession:comp
 Once the user has logged into your app, inform Streem that they are logged in:
 
 ```swift
-    Streem.sharedInstance.identify(
-        userId: "john",
-        expert: false,
-        name: "John Smith", 
-        avatarUrl: "http://..."
+    Streem.sharedInstance.login(
+            companyCode: "acme-inc", 
+            email: "john.smith600@gmail.com", 
+            password: "StreemIsC00l!", 
+            expert: true, 
```

Comment:
Can we take `expert` off the API, and hard-code the fact that logging in with username/password means you are an expect?

Reply:
@seanadkinson Are you sure we want to do that? I thought we have some future use cases where customers might have username/password logins too (e.g. Nauticon).

Reply:
@garyp I think we thought that could happen at some point, but now with invites being much easier to provide to customers, and our REST API available if they did need to access protected data, i don't think we'd actually create user accounts with credentials for customers anymore.  Even if we did do that in the future, that customer account would almost certainly be configured as such on the backend, and the provided credentials would be enough to determine if they were an expert or not.

Similar to the login/identify discussion, now that we are providing confidential information in order to login, as opposed to just being told who the current user is, I think specifying that they are an `expert` in the current world is more confusing than it is potentially helpful in the future.

Reply:
[✅](https://github.com/streem/streem-sdk-ios/pull/16/commits/9a9971e525117729bee454e3726cce8268421c07)